### PR TITLE
187372266 v3 DI allCases

### PR DESCRIPTION
**Pull request by tealefristoe**
PT Story: https://www.pivotaltracker.com/story/show/187372266

This PR adds support for `delete` and `get` `allCases` requests from plugins in CODAP v3.

Note that `delete allCases` should be undoable, but it's not working right now. However, I suspect this is a bug with `dataSet.removeCases`. I tried making deleting cases undoable via the tile menu, and that also won't work. The correct strings are being applied to the undo/redo button tooltips, so the action is going on the undo stack. It's just that pushing the buttons fails to undo the action.

### CODE REVIEW

**Review by kswenson — APPROVED** [review]
👍 Looks good -- just a couple of suggested typing changes and a question.

File: v3/src/data-interactive/handlers/all-cases-handler.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      dataContext.ungroupedAttributes
+
+    const cases = dataContext.getGroupsForCollection(collection.id)?.map((c, caseIndex) => {
+      const id = c.pseudoCase.__id__
+
+      const parent = dataContext.getParentCase(id, collection.id)?.pseudoCase.__id__
+
+      // iphone-frame was throwing an error when Array.from() wasn't used here for some reason.
+      const childPseudoCaseIds = c.childPseudoCaseIds && Array.from(c.childPseudoCaseIds)
+      const childCaseIds = c.childCaseIds && Array.from(c.childCaseIds)
```
[the remark this answers is not in the document]
Comment (tealefristoe) [reply v3/src/data-interactive/handlers/all-cases-handler.ts:31]: I'm also not sure what's going on here, and once I found something that worked I didn't dig into it any further. Here's the error I was seeing, along with the setup that caused it.

**Pull request merged**